webviews/src/views/RokuAutomationView/ConfigControls.svelte

```diff
@@ -0,0 +1,141 @@
+<script>
```

Comment:
Let's enable typescript in this script. And update all the variables declared at the top of the file to have types instead of comments.
```suggestion
<script lang="ts">
```